src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -1,5 +1,7 @@
 package core.basesyntax;
 
+//import java.lang.IllegalArgumentException;
```

Comment:
please remove this. Try not to use comments in your code

Reply:
Fixed. Thanks.